prowler/providers/aws/services/documentdb/documentdb_service.py

```diff
@@ -0,0 +1,89 @@
+import threading
+from typing import Optional
+
+from pydantic import BaseModel
+
+from prowler.lib.logger import logger
+from prowler.providers.aws.aws_provider import generate_regional_clients
+
+
+################## DocumentDB
+class DocumentDB:
+    def __init__(self, audit_info):
+        self.service = "docdb"
+        self.session = audit_info.audit_session
+        self.audited_account = audit_info.audited_account
+        self.audit_resources = audit_info.audit_resources
+        self.audited_partition = audit_info.audited_partition
+        self.regional_clients = generate_regional_clients(self.service, audit_info)
+        self.db_instances = []
+        self.__threading_call__(self.__describe_db_instances__)
```

Comment:
All this fields have been moved to a new `AWSService` class here https://github.com/prowler-cloud/prowler/pull/2638

```suggestion
class DocumentDB(AWSService):
    def __init__(self, audit_info):
        # Call AWSService's __init__
        super().__init__("docdb", audit_info)
        self.db_instances = []
        self.__threading_call__(self.__describe_db_instances__)
```